legend-engine-protocol-relational/src/main/java/org/finos/legend/engine/protocol/pure/v1/RelationalProtocolExtension.java

```diff
@@ -156,7 +156,8 @@
                                 Tuples.pair(TestDatabaseAuthenticationStrategy.class, "test"),
                                 Tuples.pair(DelegatedKerberosAuthenticationStrategy.class, "delegatedKerberos"),
                                 Tuples.pair(SnowflakePublicAuthenticationStrategy.class, "snowflakePublic"),
-                                Tuples.pair(GCPApplicationDefaultCredentialsAuthenticationStrategy.class, "gcpApplicationDefaultCredentials")
+                                Tuples.pair(GCPApplicationDefaultCredentialsAuthenticationStrategy.class, "gcpApplicationDefaultCredentials"),
```

Comment:
If we are moving to db/auth specific modules. I believe protocol classes should also move there